uaaextras/webapp.py

```diff
@@ -58,13 +58,24 @@
         "socket_connect_timeout": 0.5
     }
 
-# Connect to redis
-try:
-    r = redis.StrictRedis(**redis_env)
-    r.info()
-except Exception as exc:
-    logging.warn('Unable to connect to Redis: {0}'.format(exc))
-    r = None
+
+class RedisWrapper(object):
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+
+    @property
+    def redis(self):
+        if not hasattr(self, '_redis'):
```

Comment:
Right, but I thought the point was that this was trying to solve an issue where there was a lost connection? I could have connected once in the past, still have `_redis` as an attr, but not actually have a valid connection. Maybe I'm misunderstanding the problem?